src/roles/common/tasks/vcenter-deploy-image.yml

```diff
@@ -1,11 +1,12 @@
 - name: Deploy image on vCenter (ignoring errors) (Can take several minutes)
   connection: local
-  shell: >-
+  command: >-
     {{ ovftool_command }}
     "vi://{{ vcenter.username | urlencode }}:{{ vcenter.password | urlencode }}@{{ vcenter_path }}"
   register: deploy_result
   ignore_errors: yes
   no_log: "{{ lookup('env', 'METROAE_NO_LOG') or 'true' }}"
+  changed_when: deploy_result.rc == 0
```

Comment:
Is there a conflict between ignore_errors and the possible result of "True" in this check?